devicetypes/sky-nie/min-smart-plug-dimmer.src/min-smart-plug-dimmer.groovy

```diff
@@ -0,0 +1,672 @@
+/**
+ *      Min Smart Plug Dimmer v1.1.2
+ *
+ *  	Models: MINOSTON (MP21ZD MP22ZD/ZW39S ZW96SD)
+ *
+ *  Author:
+ *   winnie (sky-nie)
+ *
+ *	Documentation:
+ *
+ *  Changelog:
+ *
+ *    1.0.1 (06/30/2021)
+ *      - Add new product supported
+ *
+ *    1.1.1 (05/06/2021)
+ *      - 1.Solve the problem that the temperature cannot be displayed normally
+ *      - 2.Synchronize some of the latest processing methods, refer to Minoston Door/Window Sensor
+ *
+ *    1.0.1 (03/17/2021)
+ *      - Simplify the code, delete dummy code
+ *
+ *    1.0.0 (03/11/2021)
+ *      - Initial Release
+ *
+ * Reference：
+ *    https://github.com/krlaframboise/SmartThings/blob/master/devicetypes/krlaframboise/eva-logik-in-wall-smart-dimmer.src/eva-logik-in-wall-smart-dimmer.groovy
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+metadata {
+    definition (
+            name: "Min Smart Plug Dimmer",
+            namespace: "sky-nie",
+            author: "winnie",
+       //     mnmn: "SmartThings",
+       //     vid:"generic-dimmer-3",
```

Comment:
If you don't need `mnmn` and `vid` to be defined, just remove it.

Reply:
OK, I removed it.